antarest/study/storage/rawstudy/model/filesystem/root/output/simulation/mode/mcall/digest.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+import typing as t
+
+import pandas as pd
+
+from antarest.core.model import JSON
+from antarest.core.serialization import AntaresBaseModel
+from antarest.study.storage.rawstudy.model.filesystem.config.model import FileStudyTreeConfig
+from antarest.study.storage.rawstudy.model.filesystem.context import ContextServer
+from antarest.study.storage.rawstudy.model.filesystem.root.output.simulation.mode.mcall.synthesis import OutputSynthesis
+
+
+class DigestMatrixDTO(AntaresBaseModel):
+    columns: t.List[t.Union[str, t.List[str]]]
```

Comment:
The concepts of columns and rows are really related to the display: in an API, we should have more self-defined and business-related names, like `areas` for row names and `variables` for columns names. Otherwise, we don't know what the data represents.

In the same order of idea, instead of a list of string for the headers we could have a small class containing variable name, unit, and aggregation type.

We can keep this if we consider that it's really only meant to be used for the display, not by an API client, but in that case maybe we could rename the DigestDTO and the endpoint so that it's really identified as ui-related ?